spec/models/location_spec.rb

```diff
@@ -1,18 +1,47 @@
 describe Location do
+  it { is_expected.to validate_presence_of(:address) }
+
   context 'associations' do
-    it { should belong_to(:organisation) }
-    it { should have_many(:ips) }
+    it { is_expected.to belong_to(:organisation) }
+    it { is_expected.to have_many(:ips) }
   end
 
   describe "#save" do
-    let(:organisation) { create(:organisation) }
     subject { build(:location, organisation: organisation) }
-    before do
-      subject.save
-    end
+
+    let(:organisation) { create(:organisation) }
+
+    before { subject.save }
 
     it 'sets the radius_secret_key' do
       expect(subject.radius_secret_key).to be_present
     end
   end
+
+  describe '#full_address' do
+    before do
+      subject.address = address
+      subject.postcode = postcode
+    end
+
+    context 'with address' do
+      let(:address) { "121 Fictional Street" }
+
+      context 'and postcode' do
+        let(:postcode) { "FI5 S67" }
+
+        it "combines the two" do
```

Comment:
Would stick to single quotes for non interpolated strings for consistency.
Pedantic sorry!

Reply:
Not at all, consistency is <3